ups/meas_base.table

```diff
@@ -3,6 +3,7 @@ setupRequired(pipe_base)
 setupRequired(daf_base)
 setupRequired(pex_config)
 setupRequired(coadd_utils)
+setupOptional(geom)
```

Comment:
Sorry, I blindly followed the HSC entry (which I guess should have been Required).  Amended.